app/scripts/views/mainView.js

```diff
@@ -282,7 +282,10 @@ define([
 
 		showMore: function(e) {
 			e.preventDefault();
+			//this to toggle the text between read more and hide text
 			//this.text('See Less');
+			//this to taggle show/hide of the extra text
+			$('#extraText').toggle();
```

Comment:
@mkayan It recommended to use this.$el.find('#extraText').toggle() when targeting a particular div element in a view. Consider doing that if you run into any issues.